### DISCUSSION TITLE
Overview when clicking on an empty workspace.

### DISCUSSION THREAD

**Discussion by Okanda** [Favo02/workspaces-by-open-apps#56] [Ideas / Feature requests]
Hello. I'm testing the extension and I like it :)

I used to use Space Bar but with your extension I can have a better overview of opened apps. 
But I missed one feature : it is the activation of the Gnome Overview screen when choosing/clicking on an empty workspace. 

It's a killer feature ;)

Maybe we can see that it in workspaces-by-open-apps ? 

Thank you for your job. 

LCG

**Comment by Favo02** [comment]
Thanks for the feedback!

That's a great idea, I don't think it will be too hard to implement a new setting to toggle this feature.
I cannot guarantee anything but I think the next update will have this, maybe in a week or two.

One last thing: which GNOME version are you using? 44- or 45?

**Reply by Okanda** [reply]
Whaou. Thanks for your response! 

I use Gnome 45 on EndeavourOS.

**Comment by Favo02** [comment]
Added in #63, new version will be published on GNOME extensions store in a few days.
